CHANGELOG.md

```diff
@@ -5,7 +5,14 @@ project adheres to [Semantic Versioning](http://semver.org).
 
 ## [Unreleased (`master`)][unreleased]
 
-Nothing at the moment.
+### Changed
+
+- Changed project SASS variables to native CSS variables
```

Comment:
```suggestion
- Changed project SASS variables to native CSS Custom Properties
```

Should we say “Custom Properties” to match [the spec](https://drafts.csswg.org/css-variables/)?